aptos-move/aptos-gas-schedule/src/gas_schedule/misc.rs

```diff
@@ -207,8 +208,7 @@ impl ValueVisitor for AbstractValueSizeVisitor<'_> {
 
     #[inline]
     fn visit_closure(&mut self, _depth: usize, _len: usize) -> bool {
-        // TODO(#15664): introduce a dedicated gas parameter?
-        self.size += self.params.struct_;
+        self.size += self.params.closure;
```

Comment:
Also on line 383, 533 there is still the same `// TODO(#15664)` - these should be updated as well?

Reply:
Done